### PR TITLE
Login ctrl

### PR DESCRIPTION
**Pull request by IPoyda**
Login (do not throw stones)

### CODE REVIEW

File: frontend/app/scripts/controllers/login.js [hunk 1]
```diff
@@ -2,29 +2,50 @@
...
-      }
-      //TODO clear form after success
-    }
-
-    this.clearLocalStorage = function () {
-      delete $localStorage.email;
+    $scope.login = function () {
+      $scope.dataLoading = true;
+      AuthenticationService.Login($scope.credentials, function (response) {
+        if (response!=null) {
```
Comment (pashapovalov) [inline frontend/app/scripts/controllers/login.js]: So response param can be undifined. You should use

``` javascript
  if (response) {
```

This check  null and undefined.

File: frontend/app/scripts/controllers/login.js [hunk 2]
```diff
@@ -2,29 +2,50 @@
...
+          AuthenticationService.SetCredentials(response.name, response.details.sessionId, true);
+          if($rootScope.locationPath!="/login"){
```
Comment (pashapovalov) [inline frontend/app/scripts/controllers/login.js]: You should use strick check !== for compare strings.
Add also white space please.

File: frontend/app/scripts/controllers/login.js [hunk 3]
```diff
@@ -2,29 +2,50 @@
...
-      //TODO clear form after success
-    }
-
-    this.clearLocalStorage = function () {
-      delete $localStorage.email;
+    $scope.login = function () {
+      $scope.dataLoading = true;
+      AuthenticationService.Login($scope.credentials, function (response) {
+        if (response!=null) {
+          AuthenticationService.SetCredentials(response.name, response.details.sessionId, true);
```
Comment (pashapovalov) [inline frontend/app/scripts/controllers/login.js:14]: Please add check for remember me. If it no set, didn't call SetCredentials

File: frontend/app/styles/eventMenu.less [hunk 4]
```diff
@@ -28,6 +28,7 @@
 
 .ng-aside.horizontal.right.in .modal-dialog {
   width: 100%;
+  padding-bottom: 20%;
```
Comment (pashapovalov) [inline frontend/app/styles/eventMenu.less]: No percent!

File: frontend/app/styles/feedback.less [hunk 5]
```diff
@@ -1,22 +1,25 @@
...
 }
 
 .feedback-field {
-  margin-top: 3%;
+  margin-top: 1%;
 }
 
 .feedback-info {
-  padding-left: 28%;
+  margin-top: 11%;
```
Comment (pashapovalov) [inline frontend/app/styles/feedback.less]: No pecent!

File: frontend/app/styles/feedback.less [hunk 6]
```diff
@@ -1,22 +1,25 @@
...
 }
 
 .feedback-container {
-  text-align: center;
+  background-color: #E8EBED;
 }
 
 .feedback-field {
-  margin-top: 3%;
+  margin-top: 1%;
```
Comment (pashapovalov) [inline frontend/app/styles/feedback.less]: no percent!

File: src/main/java/com/exadel/filter/CORSFilter.java [hunk 7]
```diff
@@ -15,10 +15,11 @@ public void doFilter(ServletRequest req, ServletResponse res,
 			FilterChain chain) throws IOException, ServletException {
```
Comment (pashapovalov) [inline src/main/java/com/exadel/filter/CORSFilter.java:15]: Are you sure that you should add this file, too?

File: pom.xml [hunk 8]
```diff
@@ -54,6 +54,11 @@
 		</dependency>
```
Comment (pashapovalov) [inline pom.xml:54]: Are you sure that you should add this file?

File: frontend/app/scripts/controllers/login.js (hunk 3 above)
Comment (IPoyda) [inline frontend/app/scripts/controllers/login.js:14]: Current check is not needed because remember me is default functionality.

**Pull request merged**